ProductsSimpleAppFront/app/phones/editor-bar/editor-bar.component.js

```diff
@@ -0,0 +1,11 @@
+// Register editorBar component, along with its associated controller and template
```

Comment:
Na to nie zwracaj uwagi. Miałem pomysł na dodanie linku dla "administratora", ale na razie to szczątek ;)